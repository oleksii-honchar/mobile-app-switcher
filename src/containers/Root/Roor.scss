@import 'styles/variables';
@import 'styles/z-indexes';

body {
  background-color: #eff1f2;
}