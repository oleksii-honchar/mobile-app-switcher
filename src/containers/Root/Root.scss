@import 'styles/variables';
@import 'styles/z-indexes';

.o-mas-debug {
  display: flex;
  flex-direction: column;
  width: 100%;
  height: 100%;
}