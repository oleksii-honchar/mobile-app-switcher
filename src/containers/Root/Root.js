import React from 'react'

import ErrorBoundary from '@containers/ErrorBoundary'

import MobileApplicationSwitcher from '@components/MobileApplicationSwitcher/MobileApplicationSwitcher'
// import MasVanilla from '@components/MasVanilla/MasVanilla'
// import MasVanillaOne from '@components/MasVanillaOne/MasVanillaOne'

import { SwitchBar } from './SwitchBar';
import { BodyLog } from './BodyLog';

import './Root.scss'

export function Root () {
    return [
      <div className="o-mas-debug">
        <ErrorBoundary><SwitchBar/></ErrorBoundary>
        <BodyLog/>
      </div>,
      <ErrorBoundary><MobileApplicationSwitcher/></ErrorBoundary>
      // <ErrorBoundary><MasVanilla/></ErrorBoundary>
      // <ErrorBoundary><MasVanillaOne/></ErrorBoundary>
    ]
}