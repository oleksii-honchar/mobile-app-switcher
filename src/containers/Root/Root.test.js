import React from 'react'
import { render } from '@testing-library/react'

import { Root } from './Root'

describe('Root', () => {
  it('matches snapshot when no props provided', () => {
    const { container } = render(
      <Root />
    )
    expect(container).toMatchSnapshot()
  })
})
