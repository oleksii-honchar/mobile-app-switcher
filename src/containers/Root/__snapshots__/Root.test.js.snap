// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root matches snapshot when no props provided 1`] = `
<div>
  <h1>
    Hello world!
  </h1>
</div>
`;
